catalyst-gateway/bin/src/service/common/objects/account_votes.rs

```diff
@@ -35,6 +36,8 @@ pub(crate) struct AccountVote {
     /// The hex-encoded ID of the vote plan.
     pub(crate) vote_plan_id: VotePlanId,
     /// Array of the proposal numbers voted for by the account ID within the vote plan.
+    // TODO - Recheck max items and max value
```

Comment:
I'm a bit confused with this. Does this mean we need to verify that the values mentioned are correct? (i.e. you assigned the current values arbitrarily?)

Reply:
Yes, if there is one.  Otherwise I will set to the maximum of the given type (eg. u8 will be set to u8::MAX)